cli/command/image/remove_test.go

```diff
@@ -44,7 +44,7 @@ func TestNewRemoveCommandErrors(t *testing.T) {
 		},
 	}
 	for _, tc := range testCases {
-		cmd := NewRemoveCommand(test.NewFakeCli(&fakeClient{
+		cmd := NewRemoveCommand(test.NewFakeCliWithOutput(&fakeClient{
```

Comment:
Could just be `NewFakeCli`, but I know you're not trying to exhaustively convert these.